pkg/cmd/release/create/create.go

```diff
@@ -268,12 +279,20 @@ func createRun(opts *CreateOptions) error {
 		}
 	}
 
+	if opts.Draft && len(opts.DiscussionCategoryName) > 0 {
```

Comment:
This validation is redundant if the `RunE` block already performs validation of cli inputs

Reply:
I think you'd still need to perform validation after the interactive bit, because the value of opts.Draft could change. There is a survey question that starts on line 252 that lets you change that.